### DISCUSSION TITLE
Specify @time_trigger range with fixed time?

### DISCUSSION THREAD

**Discussion by ivomans** [custom-components/pyscript#542] [Q&A] [answered]
I've been searching and trying quite a lot, but cann't find the proper syntax. How should I exactly limit to a range between sunset and 01:00 at night?

Searching for something like this:
```
@mqtt_trigger("something")
@time_trigger("range(sunset, 01:00)")
```
But all I get is "Can't parse range(sunset, 01:00) in time_trigger check"

**Comment by craigbarratt** [comment, marked as answer]
If you want to limit the time range over which the `@mqtt_trigger` is active, you should use `@time_active()`.  As you discovered, `@time_trigger` doesn't take a `range` argument.
